daemon/daemon.go

```diff
@@ -975,13 +824,14 @@ func (daemon *Daemon) Mount(container *Container) error {
 	if err != nil {
 		return fmt.Errorf("Error getting container %s from driver %s: %s", container.ID, daemon.driver, err)
 	}
-	if container.basefs == "" {
-		container.basefs = dir
-	} else if container.basefs != dir {
-		daemon.driver.Put(container.ID)
-		return fmt.Errorf("Error: driver %s is returning inconsistent paths for container %s ('%s' then '%s')",
-			daemon.driver, container.ID, container.basefs, dir)
+	if container.basefs != dir {
+		if container.basefs != "" && runtime.GOOS != "windows" {
+			daemon.driver.Put(container.ID)
+			return fmt.Errorf("Error: driver %s is returning inconsistent paths for container %s ('%s' then '%s')",
+				daemon.driver, container.ID, container.basefs, dir)
+		}
```

Comment:
I wanted to leave the error case the same for Linux (graph driver returning inconsistent paths). On Windows, there are circumstances where the mounted volume path for a given disk chain can change, and these are expected, not errors. So on Windows, we instead want to accept whatever path is returned by the graph driver.